File: aandroid5pmgroup/RestapiExamples/app/src/main/java/com/example/restapiexamples/ApiClient.java
package com.example.restapiexamples;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class ApiClient {
    public static final String BASE_URL ="https://jsonplaceholder.typicode.com/";

    static Retrofit getRerorfit()
    {
        retrofit2.Retrofit retrofit = new Retrofit.Builder()
                .baseUrl(BASE_URL)
                .addConverterFactory(GsonConverterFactory.create())
                .build();

        return retrofit;
    }



}

File: Rahoul3pm/My Application/app/src/main/java/com/example/myapplication/ApiClient.kt
package com.example.myapplication

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object ApiClient {
    var BASE_URL = "https://jsonplaceholder.typicode.com/"


 fun getretrofitlbuilder():Retrofit
 {
     var retrofit = Retrofit.Builder()
         .baseUrl(BASE_URL)
         .addConverterFactory(GsonConverterFactory.create())
         .build()
     return retrofit
 }
}